src/main/scala/edu/berkeley/cs/rise/opaque/execution/EncryptedSortExec.scala

```diff
@@ -41,37 +43,39 @@ object EncryptedSortExec {
     Utils.ensureCached(childRDD)
     time("force child of EncryptedSort") { childRDD.count }
     // RA.initRA(childRDD)
+    JobVerificationEngine.addExpectedOperator("EncryptedSortExec")
```

Comment:
Can this be place somewhere else? Given a DataFrame, you can get the physical plan directly by calling `df.queryExecution`: https://stackoverflow.com/questions/41716295/what-is-the-use-of-queryexecution-in-spark-dataframe